elytron/src/main/resources/schema/wildfly-elytron_4_0.xsd

```diff
@@ -5178,4 +5179,118 @@
+        </xs:annotation>
+        <xs:sequence>
+            <xs:element name="jaspi-configuration" type="jaspiConfigurationType" maxOccurs="unbounded" />
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="jaspiConfigurationType">
+        <xs:annotation>
+            <xs:documentation>
+                An individual JASPI configuration.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:sequence>
+            <xs:element name="server-auth-modules">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="server-auth-module" type="serverAuthModuleType" maxOccurs="unbounded" />
+                    </xs:sequence>
+                </xs:complexType>
+            </xs:element>
+        </xs:sequence>
+        <xs:attribute name="name" type="xs:string" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    The name of this JASPI configuration.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="layer" type="xs:string" default="*">
+            <xs:annotation>
+                <xs:documentation>
+                    The layer this configuration should be associated with.
+
+                    If set to '*' this configuration will be associated with all layers and resolved according the the
+                    resolution rules defined within the JSR-196 specification.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="application-context" type="xs:string" default="*">
+            <xs:annotation>
+                <xs:documentation>
+                    The application context this configuration should be associated with.
+
+                    If set to '*' this configuration will be associated with all application contexts and resolved according the the
+                    resolution rules defined within the JSR-196 specification.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="description" type="xs:string">
+            <xs:annotation>
+                <xs:documentation>
+                    Descrption for this JASPI configuration.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+    
+    <xs:complexType name="serverAuthModuleType">
+        <xs:sequence>
+            <xs:element name="options" minOccurs="0">
+                <xs:annotation>
+                    <xs:documentation>
+                        Configuration options to be passed into the ServerAuthModule during initialisation.
+                    </xs:documentation>
+                </xs:annotation>
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="property" type="propertyType" maxOccurs="unbounded" />
```

Comment:
@darranl In the model this looks like it is called 'options', and since you are using PersistentResourceXMLDescription I think that is what the parser expects?